distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -85,9 +53,12 @@ Before the ${ddf-solr} Application can be installed:
 
 ===== Installing
 
-. Navigate to the ${ddf-solr} application in the Admin console.
-. Under the *Features* tab, uninstall the `catalog-solr-external-provider` feature.
-. Install the `catalog-solr-embedded-provider`
+. Edit the <INSTALLATION_DIR>/etc/system.properties
+
+.. Comment out the Solr Client Configuration for *Cloud Solr Client* and *Embedded Solr Client*.
```

Comment:
Should be _Http Solr Client_ instead of _Embedded Solr Client_.

We may also want to add the word _sections_ at the end of the sentence and also on L28.